packages/design-system/src/styles/settings/variables/_forms.scss

```diff
@@ -25,7 +29,111 @@ $choice-border-radius: 0 !default;
 $choice-size: $spacer-4 !default;
 $choice-size-small: 20px !default;
 
+// Buttons
+$button-line-height: 1.3 !default;
 $button-border-width: 1px !default;
+$button-font-weight: $font-bold !default;
```

Comment:
Added font weight to be used at the mgov level - they don't use bold for their button text